metricbeat/magefile.go

```diff
@@ -126,9 +126,8 @@ func GoTestIntegration(ctx context.Context) error {
 // not supported.
 func customizePackaging() {
 	var (
-		archiveModulesDir   = "modules.d"
-		linuxPkgModulesDir  = "/usr/share/{{.BeatName}}/modules.d"
-		darwinDMGModulesDir = "/Library/Application Support/{{.BeatVendor}}/{{.BeatName}}/modules.d"
```

Comment:
Is it in purpose that now the Mac files are placed under a different location?

Reply:
The new MacOS installer puts our typical `path.home` files into `/Library/Application Support` because this is conventional (it's like `/usr/share`).

The config and logs go to `/etc` and `/var `just like Linux. So the `modules.d` was wrong for MacOS too.